packages/components/atoms/f-button/stories/IconButton.stories.js

```diff
@@ -1,88 +1,127 @@
+                :href="href"
+                :isIcon="true"
+                :isLoading="isLoading">
+                <PlusIcon />
+                <span class="is-visuallyHidden">
+                    {{ icon.text }}
+                </span>
+            </f-button>
 
-        <f-button
-            class="u-spacingRight u-spacingBottom--large"
-            :buttonType="buttonType"
-            :buttonSize="buttonSize"
-            :actionType="actionType"
-            :href="href"
-            :isIcon="true"
-            :isLoading="isLoading">
-            <MoreVerticalIcon />
-        </f-button>
+            <f-button
+                class="u-spacingBottom--large"
+                v-for="(icon, index) in list"
+                :key="index"
+                :buttonType="icon.type"
+                :buttonSize="icon.size"
+                :actionType="actionType"
+                :href="href"
+                :isIcon="true"
+                :isLoading="isLoading">
+                <MoreVerticalIcon />
+                <span class="is-visuallyHidden">
+                    {{ icon.text }}
+                </span>
+            </f-button>
+        </div>
     </div>`
 });
+IconButtonComponent.argTypes = {
+    isLoading: SharedButtonArgTypes.isLoading,
+    href: SharedButtonArgTypes.href,
+    actionType: {
+        control: { type: 'select', options: ['button', 'submit', 'reset'] },
+        description: 'Choose the action type of the button',
+        defaultValue: 'button'
+    }
+};
+
+IconButtonComponent.args = {
+    iconButtonLists: [
+        [
+            { type: 'primary', size: 'large', text: 'Large Primary Icon Button' },
+            { type: 'primary', size: 'medium', text: 'Medium Primary Icon Button' },
+            { type: 'primary', size: 'small', text: 'Small Primary Icon Button' },
+            { type: 'primary', size: 'xsmall', text: 'Xsmall Primary Icon Button' }
+        ],
+        [
+            { type: 'secondary', size: 'large', text: 'Large Secondary Icon Button' },
+            { type: 'secondary', size: 'medium', text: 'Medium Secondary Icon Button' },
+            { type: 'secondary', size: 'small', text: 'Small Secondary Icon Button' },
+            { type: 'secondary', size: 'xsmall', text: 'Xsmall Secondary Icon Button' }
+        ],
+        [
+            { type: 'ghost', size: 'large', text: 'Large Ghost Icon Button' },
+            { type: 'ghost', size: 'medium', text: 'Medium Ghost Icon Button' },
+            { type: 'ghost', size: 'small', text: 'Small Ghost Icon Button' },
+            { type: 'ghost', size: 'xsmall', text: 'Xsmall Ghost Icon Button' }
+        ],
+        [
+            { type: 'ghostTertiary', size: 'large', text: 'Large Ghost Tertiary Icon Button' },
+            { type: 'ghostTertiary', size: 'medium', text: 'Medium GhostTertiary Icon Button' },
+            { type: 'ghostTertiary', size: 'small', text: 'Small GhostTertiary Icon Button' },
```

Comment:
Ultra nit, feel free to update another time
```suggestion
            { type: 'ghostTertiary', size: 'small', text: 'Small Ghost Tertiary Icon Button' },
```